endpoints/parameter-forwarding.md

```diff
@@ -1,126 +1,181 @@
 ---
-lastmod: 2018-09-27
+lastmod: 2018-11-27
 date: 2018-07-20
 aliases:
 - /docs/features/parameter-forwarding/
-toc: true
 linktitle:  Parameter forwarding
 title: Parameter forwarding
 weight: 30
 menu:
   documentation:
     parent: endpoints
 ---
-KrakenD always **alleviates backends** avoiding to pollute them with everything coming from the clients. By default, **no parameters sent by clients are forwarded to backends** and if needed they will require an explicit declaration in the configuration file.
+KrakenD is an API Gateway, and when it comes to forward query strings, cookies, and headers, it **does not behave like a regular proxy** by forwarding parameters to the backend.
 
-The parameter forwarding refers to:
+The **default policy** for data forwarding works as follows:
 
-- [Query string](#query-string-forwarding)
-- [Headers](#headers-forwarding)
-- [Cookies](#cookies-forwarding)
+- No [query string](#query-string-forwarding) parameters are forwarded to the backend
+- No [headers](#headers-forwarding) are forwarded
+- No [cookies](#cookies-forwarding) are forwarded
+
+You can change this behavior according to your needs, and define which elements are allowed to pass.
+
+However, if you want a specific endpoint to act just like a regular proxy without doing any operations use the `no-op` option on its encoding options. When using `no-op`, the gateway can't do anything with the content. The request goes from the client to the backend "as is".
```

Comment:
this is not accurate. the `no-op` encoding just applies to the bodies of the request and the response. It does not affect the header nor url propagation